Framework/Kernel/test/CMakeLists.txt

```diff
@@ -23,7 +23,7 @@ if ( CXXTEST_FOUND )
 )
   add_dependencies ( FrameworkTests KernelTest )
   # Test data
-  add_dependencies ( KernelTest StandardTestData )
+  #add_dependencies ( KernelTest StandardTestData )
```

Comment:
Is this supposed to be commented out?

Reply:
Nice catch! That absolutely should not be commented out.